en/firefox/features/shared.ftl

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/features/
+
+features-shared-a-better-internet-experience = A better internet experience
+
+features-shared-browse-faster = Browse faster
+features-shared-your-favorite-extensions = Your favorite extensions
+features-shared-balanced-memory = Balanced memory
```

Comment:
should this be `Balanced memory usage`?

Reply:
That's the old string, "Balanced memory" is the new one.